guide/english/bootstrap/jumbotron/index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Jumbotron
+---
+A jumbotron indicates a big grey box for calling extra attention to some special content or information.
+
+Use a <div> element with class jumbotron to create a jumbotron
+
+**Code Example:**
+
+`<div class="jumbotron">
+  <h1>Jumbotron</h1> 
+  <p>This is a jumbotron sample</p> 
+</div>`
+
+<div class="jumbotron">
+  <h1>Jumbotron</h1> 
+  <p>This is a jumbotron sample</p> 
+</div>
+
+If you want a full-width jumbotron without rounded borders, add the .jumbotron-fluid class and a .container or .container-fluid inside of it:
+
+For full-width jumbotron withour rounded borders, add class jumbotron-fluid and a container or container fluid inside the div
+
+`<div class="jumbotron jumbotron-fluid">
+  <div class="container">
+    <h1>Jumbotron</h1> 
+    <p>This is a jumbotron sample</p> 
+  </div>
+</div>`
+
+<div class="jumbotron jumbotron-fluid">
+  <div class="container">
+    <h1>Jumbotron</h1> 
+    <p>This is a jumbotron sample</p> 
+  </div>
+</div>
+
+#### More Information: 
+* <a href='https://www.w3schools.com/bootstrap4/bootstrap_jumbotron.asp' target='_blank' rel='nofollow'>Jumbotron</a>
```

Comment:
Please try to link to an MDN resource on Jumbotron instead of w3schools, if possible.